BranchSDK/src/BranchIO/SessionInfo.h

```diff
@@ -47,13 +47,13 @@ class BRANCHIO_DLL_EXPORT SessionInfo : public PropertyManager {
     virtual bool hasSessionId() const;
 
  private:
-    static constexpr const char *const StoragePrefix = "session";
+    static constexpr const char *const SessionStorage = "session";
```

Comment:
variable that starts with a capital letter

Reply:
Fixed.